Apps/SerialConsole/tactility.py

```
@@ -287,15 +270,32 @@ def get_manifest_target_platforms(manifest, requested_platform):
-    else:
+    sdk_index_filepath = os.path.join(sdk_root_dir, "index.json")
+    if verbose:
+        print(f"Downloading {sdk_index_url} to {sdk_index_filepath}")
+    if not download_file(sdk_index_url, sdk_index_filepath):
+        # TODO: 404 check, print a more accurate error
+        print_error(f"Failed to download SDK version {version}. Check your internet connection and make sure this release exists.")
+        return False
+    sdk_index_json_file = open(sdk_index_filepath)
+    sdk_index_json = json.load(sdk_index_json_file)
+    sdk_platforms = sdk_index_json["platforms"]
+    if not platform in sdk_platforms:
```

Ruff (0.14.10)

Test for membership should be `not in`

Convert to `not in`

(E713)